HelpMyStreetFE/HelpMyStreetFE/Controllers/AccountController.cs

```diff
@@ -145,7 +145,8 @@ public async Task<IActionResult> OpenRequests()
             var viewModel = GetAccountViewModel<List<Job>>(currentUser);
             viewModel.CurrentPage = MenuPage.OpenRequests;
             
-            viewModel.PageModel = await _requestService.GetOpenJobs("", 0.0);
+            //viewModel.PageModel = await _requestService.GetOpenJobs("", 0.0);
+            viewModel.PageModel = await _requestService.GetJobsByFilterAsync(currentUser.PostalCode, 20);
```

Comment:
The sorting will now not work which was being done in "GetOpenJobs".

In order to keep the sorting change it to:

```suggestion
            viewModel.PageModel = await _requestService.GetOpenJobs(currentUser.PostalCode, 20);
```

Also the number "20" should really be in config somewhere?